### DISCUSSION TITLE
Error: can't restore cause system.replicas entries already exists

### DISCUSSION THREAD

**Discussion by nnor1** [Altinity/clickhouse-backup#939] [Q&A] [answered]
Hello everyone and thank you for your support,

we always get an error during the restore. But I'm not sure if this leads to a problem, because otherwise everything works. There is also no error in the system.errors table.

Short test scenario:
1. Backup was created
2. Database was deleted
3. Backup is imported
4. Error occurs during the restore
5. Database with all tables was restored
6. Sending of events works

Current configuration:
- Use of Clickhouse with 3 replicas (no shards)
- daily backups to s3 (thanks for the tool)
- Backup create command --> clickhouse-backup create backup_test
- Backup restore command --> clickhouse-backup restore backup_test

How do I work around/fix this error?

Thanks

![Error](https://github.com/Altinity/clickhouse-backup/assets/153502948/d5c3e104-be75-4a71-a3e8-825a2a52a73f)

**Comment by Slach** [comment, marked as answer]
error message means another replica is present and insertion in another replica continues
to avoid data duplication we can't restore

> Database was deleted

when you delete database in one replica
it stay on other replicas

use
`DROP DATABASE dbname ON CLUSTER 'cluster-name' SYNC`
and look to https://github.com/Altinity/clickhouse-backup/blob/master/Examples.md#restore

during restore you need restore schema on each replica in each shard
and restore data only on one replica in each shard

**Reply by nnor1** [reply]
Thank you
